src/distribution_spatial.cpp

```diff
@@ -228,36 +223,22 @@ MeshSpatial::MeshSpatial(pugi::xml_node node)
                     "not match the number of entities in mesh {} ({}).",
           strengths.size(), mesh_id, n_bins));
     }
-    mesh_strengths_ = std::move(strengths);
   }
-
-  if (get_node_value_bool(node, "volume_normalized")) {
+ 
+ if (get_node_value_bool(node, "volume_normalized")) {
     for (int i = 0; i < n_bins; i++) {
-      mesh_strengths_[i] *= mesh()->volume(i);
+      strengths[i] *= mesh()->volume(i);
     }
   }
 
-  total_strength_ =
-    std::accumulate(mesh_strengths_.begin(), mesh_strengths_.end(), 0.0);
+  elem_idx_dist_ = UPtrDist {new Discrete {strengths.data(), n_bins}};
```

Comment:
Happy to change it, but I was copying the pattern elsewhere in this file. (Possibly outdated?)

Reply:
I can't say as I understand the nuances as to why, but every single time a `UPtrDist` is defined, it uses this curly braces construct, across multiple files.  It may be related to the definition of `UPtrDist`:
`using UPtrDist = unique_ptr<Distribution>;`